fsnotify.go

```diff
@@ -62,5 +62,5 @@ func (e Event) String() string {
 	return fmt.Sprintf("%q: %s", e.Name, e.Op.String())
 }
 
-// Common errors that can be reported by a watcher
```

Comment:
I believe this comment is meant for multiple errors. Could you change the comment to `ErrEventOverflow can be reported by a watcher.`